classes/class-p4-post-campaign.php

```diff
@@ -400,5 +401,14 @@ public function enqueue_admin_assets() {
 			wp_register_style( 'cmb-style', get_template_directory_uri() . '/assets/css/campaign.css' );
 			wp_enqueue_style( 'cmb-style' );
 		}
+
+		/**
+		 * Hide the menu item until the product owner is happy to share it.
+		 * Related Jira issue: PLANET-3425
+		 */
+		public function remove_campaigns_menu() {
+		    remove_menu_page( 'edit.php?post_type=campaign' );
```

Comment:
It seems you have mixed indentation here. Both tabs and spaces.

Reply:
This code is not there anymore, as I followed the method that Kyriakos suggested